client/src/locale/en.js

```diff
@@ -1586,33 +1586,33 @@ const en = {
         serviceConnectionRequest: {
             actions: "",
             resend: "",
-            service: "Service",
+            service: "Application",
             requester: "Requester",
             created_at: "Created",
             message: "Message"
         },
-        actionsTooltip: "Delete the service Connection request. The request can henceforth not be approved or denied anymore.",
-        serviceConnectionRequests: "Service Connection Requests",
-        serviceConnectionRequestDeleted: "Deleted request to connect service {{service}} to collaboration {{collaboration}}.",
-        serviceConnectionRequestResend: "Resent request to connect service {{service}} to collaboration {{collaboration}}.",
-        serviceRestrictedInfo: "This collaboration is service restricted. Only the platform admin can configure services to be used by this collaboration.",
-        serviceDeleteConfirmation: "Are you sure you want to remove this service from collaboration {{collaboration}}?",
-        serviceConnectionRequestDeleteConfirmation: "Are you sure you want to retract this service connection request?",
-        serviceConnectionRequestResendConfirmation: "Are you sure you want to resend this service connection request?",
+        actionsTooltip: "Delete the application connection request. The request can henceforth not be approved or denied anymore.",
+        serviceConnectionRequests: "Application connection Requests",
```

Comment:
Application connection requests? Application Connection Requests?

Reply:
Also see above use of "application connect request" 

let's standardize on one